.gitignore

```diff
@@ -23,4 +23,6 @@ cache/
 # ignore unwanted output from presigning runbooks
 tmp-*.json
 draft*signer*.json
-ready-*
\ No newline at end of file
+ready-*
+
+records/
```

Comment:
We actually recently [changed](https://github.com/ethereum-optimism/superchain-ops/pull/459/files#diff-bac743fe3d899998e32393af53af8cc7d28c89c3d7fabfa48b01361cf8d42d9bL10) our `foundry.toml` to just use the regular `broadcast` default, so this files diff should no longer be needed